src/eq_schema/Question.js

```diff
@@ -39,22 +39,24 @@ class Question {
         return answer;
       })
     );
-    return answerArray.map(answer => new Answer(answer));
+    return answerArray.map(answer => new Answer(answer, ctx));
   }
 
-  buildDateRangeAnswers({ id, label, secondaryLabel }) {
+  buildDateRangeAnswers({ id, label, secondaryLabel }, ctx) {
     return [
       new Answer({
         label,
         type: "Date",
         id: `${id}from`,
-        mandatory: true
+        mandatory: true,
+        ctx
```

Comment:
Probably best to keep `answer` and `ctx` as separate arguments. You probably meant this?

```js
new Answer({
  label,
  type: "Date",
  id: `${id}from`,
  mandatory: true,
}, ctx);
```